src/js/shortcuts.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+import { toggleFullScreen  } from './utils.js';
+import { defaultShortcuts, keyboardShortcuts } from './store.js';
+import { derived, get } from 'svelte/store';
+
+// key -> shortcut
+export const shortcutMap = derived(keyboardShortcuts, $shorts => {
+  const reverseKeyAction = ([action, key]) => [key, action];
+
+  const existingShortcuts = Object.entries($shorts).map(reverseKeyAction);
+  const newShortcuts = Object.entries(defaultShortcuts)
+    .filter(([_key, action]) => $shorts[action] == null)
+    .map(reverseKeyAction);
+
+  return new Map([...existingShortcuts, ...newShortcuts]);
+});
```

Comment:
`existingShortcuts` and `newShortcuts` being named the exact reverse of what one would expect is kind of confusing (`existingShortcuts` is the user-set values while `newShortcuts` is the default shortcuts).

This also currently overwrites user-set keys with the default keys, so one can't use a key that was already set for something else in the default keys.

Reply:
Perhaps I should have named `existingShortcuts` as `savedShortcuts` to avoid confusion. Also, `newShortcuts` are just shortcuts that were added in a new update and weren't in the extension storage.

Reply:
Addressed in cbd9d0b.